samples/tab-deeplink/csharp/DeepLinkBot/Views/Home/ExtendedDeepLinks.cshtml

```diff
@@ -0,0 +1,311 @@
+      font-family: "Segoe UI","Apple Color Emoji","Segoe UI Emoji","Segoe UI Web";
+      font-size: 14px;
+  }
+
+  .card {
+      box-shadow: 0 4px 8px 0 rgb(0 0 0 / 20%);
+      width: 27rem;
+      min-height: 18rem;
+      overflow: hidden;
+      margin-left: 2rem;
+      margin-top: 0.5rem;
+  }
+
+  .card-container-div {
+      padding-bottom: 2rem;
+      display: flex;
+  }
+
+  .card-header
+  {
+      margin-left: 1rem;
+      margin-right: 1rem;
+      margin-top: 1rem;
+      padding-top: 0.5rem;
+      font-size: 16px !important;
+  }
+
+  button {
+      margin: 1rem 8.5rem 1rem 11.5rem;
+  }
+
+  .form-container {
+      margin-left: 1rem;
+  }
+
+  .card-body {
+      margin-left: 0.5rem;
+  }
+</style>
+
+<script type="text/javascript">
+  microsoftTeams.app.initialize();
+
+  function selectPeople() {
+      microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+          if (people) {
+              document.getElementById("Attendee").value = (people.map((p) => p.email)).join(",");
+          }
+      }).catch((error) => {
+          console.log(error);
+      });
+  }
+
+  function onCallDeepLinkButtonClick(callModalities) {
+      microsoftTeams.people.selectPeople({ setSelected: [], openOrgWideSearchInChatOrChannel: true, singleSelect: false }).then((people) => {
+          if (people) {
+              if(microsoftTeams.call.isSupported()) {
+                  microsoftTeams.call.startCall({ targets: people.map((p) => p.email),
+                      requestedModalities: [callModalities]})
+                  .then((result) => {
+                      console.log(result);
+                  }).catch((error) => {
+                          console.log(error);
+                  });
+              }
+              else {
+                  alert("Call isn't supported");
+              }
+          }
+      }).catch((error) => {
+          console.log(error);
+      });
+  }
+
+  function onSchedulingDialogClick() {
+      var subjectInput = document.getElementById("Subject");
+      var startTimeInput = document.getElementById("StartTime");
+      var endTimeInput = document.getElementById("EndTime");
+      var contentInput = document.getElementById("Content");
+      var attendeeInput = document.getElementById("Attendee");
+
+      if (subjectInput.value.trim() !== "" && startTimeInput.value !== "" && endTimeInput.value !== "" && attendeeInput.value !== "") {
+          var meetingDetails =
+          {
+              attendees: attendeeInput.value,
+              content: contentInput.value.trim(),
+              endTime: endTimeInput.value,
+              startTime: startTimeInput.value,
+              subject: subjectInput.value.trim(),
+          };
+
+          //microsoftTeams.calendar.composeMeeting(meetingDetails).
+          //    then((result) => { console.log(result); }).
+          //    catch((error) => { console.log(error); });
```

Comment:
We can remove commented code.

Reply:
It is required there. Added another line after discussing with Wajeed - That this feature is not yet supported in Microsoft Teams.